src/main/java/com/jcabi/http/wire/LastModifiedCachingWire.java

```diff
@@ -49,162 +41,14 @@
  * @version $Id$
  * @since 1.15
  */
-public final class LastModifiedCachingWire implements Wire {
-
-    /**
-     * Last-Modified header name.
-     */
-    public static final String LAST_MODIFIED = "Last-Modified";
-
-    /**
-     * If-Modified-Since header name.
-     */
-    private static final String IF_MODIFIED_SINCE = "If-Modified-Since";
-
-    /**
-     * Cache.
-     */
-    private final transient Map<Request, Response> cache;
-
-    /**
-     * Original wire.
-     */
-    private final transient Wire origin;
+public final class LastModifiedCachingWire
+        extends AbstractHeaderBasedCachingWire {
```

Comment:
@gmandnepr indentation should be no more than 4 spaces more compared to previous line